db/migrate/20190509125709_fix_missing_remote_photo_fields.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class FixMissingRemotePhotoFields < ActiveRecord::Migration[5.1]
+  def up
+    Photo.where(remote_photo_path: nil).each do |photo|
+      photo.write_attribute(:unprocessed_image, photo.read_attribute(:processed_image))
+      photo.update_remote_path
```

Comment:
Oh, just something that came to my mind now, should we add a migration model here with that method, in case we remove it from the real model but the migration still exists?